scripts/main_obs_prep.sh

```diff
@@ -1,31 +1,58 @@
 #!/bin/bash -l
+
 echo 'main_obs_prep.sh starts'
 echo ${ROOT_GODAS_DIR}
 echo CDATE is $CDATE
 
-ObsRunDir=$RUNCDATE/Data/${CDATE}    #Path for observations to be ingested by DA
-mkdir -p ${ObsRunDir}
-
 # Prep absolute dynamic topography obs
 source ${ROOT_GODAS_DIR}/scripts/adt_prep_obs.sh 
+echo "ADT done"
 
 # Prep T&S profile obs
 source ${ROOT_GODAS_DIR}/scripts/fnmoc_prep_obs.sh
+echo "FNMOC done"
 
 # Prep ice concentration obs
-source ${ROOT_GODAS_DIR}/scripts/icec_prep_obs.sh
+#source ${ROOT_GODAS_DIR}/scripts/icec_prep_obs.sh
 
 # Prep sst obs
 ListOfSST="sst.windsat_l3u.ghrsst \
            sst.gmi_l3u.ghrsst \
            sst.amsr2_l3u.ghrsst
-           sst.avhrrmta_l3u.nesdis \
-           sst.avhrr19_l3u.nesdis \
            sst.viirs_l3u.nesdis "
 
 for SSTsource in $ListOfSST;do
-   ${ROOT_GODAS_DIR}/scripts/sst_prep_obs.sh \
-                     -i ${SSTsource}
+    sat=''
+    echo SSTsource is $SSTsource
+    echo sat is $sat
+    source ${ROOT_GODAS_DIR}/scripts/sst_prep_obs.sh \
+                     -i ${SSTsource} \
+                     -d $sat
+    mv $OUTDIR/ioda.${SSTsource}.${sat}.${DA_SLOT_LEN}h.nc \
+       $OUTDIR/ioda.${SSTsource}.${DA_SLOT_LEN}h.nc  
+    echo $SSTsource done
+done
+
+# AVHRR data
+SSTsource="sst.avhrr_l3u.nesdis"
+for sat in AVHRR19 AVHRRMTA; do
+   echo "sat is" $sat
+   source ${ROOT_GODAS_DIR}/scripts/sst_prep_obs.sh \
+                     -i  ${SSTsource} \
+                     -d  $sat
+   mv $OUTDIR/ioda.${SSTsource}.${sat}.${DA_SLOT_LEN}h.nc \
+      $OUTDIR/ioda.sst.${sat}_l3u.nesdis.${DA_SLOT_LEN}h.nc  
+# Change filename to lowercase
+   cd $OUTDIR
+   y=ioda.sst.${sat}_l3u.nesdis.${DA_SLOT_LEN}h.nc 
+   lc=`echo $y  | tr '[A-Z]' '[a-z]'`
+   if [ $lc != $y ]; then
+      mv $y $lc
+   fi
+   echo $lc
```

Comment:
Not necessary

Reply:
Will remove them but were there as a test.

Reply:
mv $y $lc
is neccessary.
Since lc=`echo $y  | tr '[A-Z]' '[a-z]' only displays the lowercase converted filename.

Reply:
The lines of code from 25 to 52, except 49 and 51, are necessary for the code to read the AVHRR19 and AVHRRMTA files to be read by the code.